_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -0,0 +1,114 @@
+---
+id: aaa1bf
+name: auto-play audio does not exceed 3 seconds
+rule_type: atomic
+description: |
+  auto-play audio content does not output audio for more than 3 seconds
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML [`audio`](https://html.spec.whatwg.org/multipage/media.html#the-audio-element), [`video`](https://html.spec.whatwg.org/multipage/media.html#the-video-element) or [`source`](https://html.spec.whatwg.org/multipage/embedded-content.html#the-source-element) elements, with a [`src`](https://html.spec.whatwg.org/multipage/media.html#attr-media-src) attribute referencing content with a duration of more than 3 seconds, that has an [`autoplay`](https://html.spec.whatwg.org/multipage/media.html#attr-media-autoplay) attribute equal to `true`, and that has both [`paused`](https://html.spec.whatwg.org/multipage/media.html#dom-media-paused) and [`muted`](https://html.spec.whatwg.org/multipage/media.html#attr-media-muted) attributes equal to `false`.
+
+## Expectation
+
+For each test target the total audio output does not last more than 3 seconds.
+ 
+## Assumptions
+
+*There are currently no assumptions*
```

Comment:
You're assuming that the source element is a child of video / audio. Should probably just solve for that instead of put it in the assumptions.

Reply:
Added to applicability